server/app/repository/AccountRepository.java

```diff
@@ -163,6 +164,38 @@ public CompletionStage<Void> updateApplicant(ApplicantModel applicant) {
         executionContext);
   }
 
+  public void updateClientDob(String newDob, ApplicantModel applicant) {
```

Comment:
question (blocking): Do we need to add new tests in `AccountRepositoryTest` that tests these methods?

Reply:
added..

Reply:
Awesome thanks!!